osmosis-1/osmosis-1.assetlist.json

```diff
@@ -10643,6 +10643,48 @@
         "osmosis-info",
         "osmosis-price:ibc/9BBA9A1C257E971E38C1422780CE6F0B0686F0A3085E2D61118D904BFE0F5F5E:1120"
       ]
+    },
+    {
+      "description": "The native staking token of the Function X",
+      "denom_units": [
+        {
+          "denom": "ibc/2B30802A0B03F91E4E16D6175C9B70F2911377C1CAE9E50FF011C821465463F9",
```

Comment:
it is incorrect to have 2 denom units with the same exponent (FX should be an alias). Please remove all edits of osmosis-1.assetlist.json the PR

Reply:
resolved